src/Appwrite/Platform/Modules/Sites/Http/Variables/Update.php

```diff
@@ -52,12 +53,13 @@ public function __construct()
             ->param('variableId', '', new UID(), 'Variable unique ID.', false)
             ->param('key', null, new Text(255), 'Variable key. Max length: 255 chars.', false)
             ->param('value', null, new Text(8192, 0), 'Variable value. Max length: 8192 chars.', true)
+            ->param('secret', false, new Boolean(), 'Is secret? Secret variables can only be updated or deleted, they cannot be read.', true)
```

Comment:
```suggestion
            ->param('secret', null, new Boolean(), 'Is secret? Secret variables can only be updated or deleted, they cannot be read.', true)
```

Reply:
If I change it to `null`, then I'll need to mark it as a required param. Would it be a breaking change for functions, if we decide to introduce the feature in functions?

Reply:
I see ky and value are both null by default, and value is optional.
Please double-check, but I think "null" means "keep as is".

I suggested the edit because few lines below, in action, I noticed you did type `?bool`, which hints that it can be null.